templates/patterns/timeline/pattern-timeline.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Timeline pattern.
+ *
+ */
+#}
+{% set _toggle_collapsed = toggle_collapsed|default("Show more %d items") %}
+{% set _toggle_expanded = toggle_expanded|default("Hide %d items") %}
+{% include '@oe-bcl/bcl-timeline/timeline.html.twig' with {
+  toggle_collapsed: _toggle_collapsed,
+  toggle_expanded: _toggle_expanded,
+  hide: {
+    from: hide_from,
+    to: hide_to|default(-2),
```

Comment:
why default -2 ?

Reply:
because it's how it's set on netlify. https://oelibrary.netlify.app/default/?path=/story/paragraphs-timeline--default

Reply:
ok will set to 0 as is on oe_theme, thank you.